docs/testbed/README.testbed.SnappiTests.md

```diff
@@ -0,0 +1,399 @@
+# Running Snappi Tests and Snappi Configuration
+
+### Author
+
+| Version | Name          | Role          |
+| ------- | ------------- | ------------- |
+| 1.0     | Dev Ojha      | Spec Guide    |
+
+
+## 1 Physical Topology
+
+### 1.1 Purpose
+
+Each DUT front-panel port is cabled 1-to-1 to a traffic-generator port of equal speed so that Snappi test scripts can exercise link-level features and inject/measure traffic end-to-end.
+
+### 1.2 Sample link map
+
+| StartDevice | StartPort | EndDevice | EndPort | Bandwidth (Mb/s) | VLAN ID | VLAN Mode |
+| ----------- | --------- | --------- | ------- | ---------------- | ------- | --------- |
+| `dut1`      | Ethernet1 | `tgen1`   | P1      | 100 000          |         | Access    |
+| `dut1`      | Ethernet2 | `tgen1`   | P2      | 100 000          |         | Access    |
+
+The CSV file that carries this information is called **`sonic_tgen_links.csv`** and must reside next to the Ansible playbooks (`/ansible` directory) so that `conn_graph_facts.py` can feed the tests.
+
+#### Topology Rules
+* Snappi tests treat every DUT front-panel port listed in `sonic_tgen_links.csv` as *test-traffic* capable; they do **not** distinguish between traditional “uplink” and “down-link” roles.
+* Any port that is cabled to the traffic generator **must** appear in the CSV; ports omitted from the file are ignored by the test harness.
```

Comment:
Currently, there is no sonic_tgen_links.csv under ansible, directory. Could you check?